index.html

```diff
@@ -44,5 +51,141 @@ <h3>I’am a Software Developer</h3>
+        sign-ups required. has been the industry's standard dummy text ever
+        since the 1500s, when an unknown printer took a standard dummy text.
+      </p>
+      <ul>
+        <li>css</li>
+        <li>html</li>
+        <li>bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <button>See Project</button>
+      <h4></h4>
+    </section>
+    <section class="marg">
+      <div class="art">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button>See Project</button>
+        <div class="art">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+        <div class="art">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+        <div class="art">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+        <div class="art">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+        <div class="art">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <section class="aboutMe">
+      <h1>About me</h1>
```

Comment:
- [ ] Kindly change this to be any heading from `h2` to `h6`, for example: `h2` since the `h1` should only exist once on your page for the main heading.